common/shared.c

```diff
@@ -1021,6 +1021,9 @@ ssize_t ft_rx(struct fid_ep *ep, size_t size)
 	}
 	/* TODO: verify CQ data, if available */
 
+	/* Ignore the size arg. Otherwise posted receive wouldn't have enough space
+	 * when the benchmark tests increments the send data size. Instead we post
+	 * a size of the entire receive buffer. */
```

Comment:
Oh, I'm just adding a comment here. This does not change any code. I spent some time wondering why we are not using the size arg when posting receive and then added this comment for future reference.